external/src/main/java/org/hawkular/alerts/AlertsStandalone.java

```diff
@@ -56,6 +57,10 @@
     @Inject
     PoliciesHistoryService policiesHistoryService;
 
+    // The triggers' lifecycle is no longer used by policies-ui-backend. It has been replaced with a Postgres table.
+    @ConfigProperty(name = "engine.triggers-lifecycle.disabled", defaultValue = "false")
+    boolean triggersLifecycleDisabled;
+
     //    @ConfigProperty(name = "engine.backend.ispn.reindex", defaultValue = "false")
```

Comment:
not related to your PR, but why is this uncommented?

Reply:
I have no idea.

Reply:
Git blame on that line leads to https://github.com/RedHatInsights/policies-engine/commit/b2ec21832890ff90787ef3b4fcb525c44d621f33